.github/workflows/quantization-periodic.yml

```diff
@@ -0,0 +1,55 @@
+name: quantization-periodic
+
+on:
+  push:
+    tags:
+      - ciflow/quantization-periodic/*
```

Comment:
I'm surprise that this new `ciflow` tag works https://github.com/pytorch/pytorch/actions/runs/17780030552,  I thought that it needs to be added to https://github.com/pytorch/pytorch/blob/main/.github/pytorch-probot.yml first